tests/test_marshal_types_dates.py

```diff
@@ -241,6 +241,26 @@ class Foo(proto.Message):
     assert Foo.pb(foo).ttl.seconds == 120
 
 
+def test_duration_write_string_nested():
```

Comment:
nit: for clarity, I suggest:
s/`another_field`/`foo_field`/g
s/`some_field`/`bar_field`/g

Reply:
Fixed in fcb5bb26557c714fd14724fa541bdfc15219a958